cypress/support/nodeManager.ts

```diff
@@ -6,7 +6,7 @@ import {
 
 const nodeManagerPlugin = createNodeManagerPlugin();
 
-const startAndOnboardNode = (
+export const startAndOnboardNode = (
```

Comment:
Do we still need to export this?

Reply:
Fixed in https://github.com/radicle-dev/radicle-upstream/pull/1682/commits/a73c2ff28bfaf69978ad91eeb60a6f440bd464bc